### ISSUE TITLE
Dns over HTTPS - Do not recommed Cloudfare

### ISSUE THREAD

**Issue by privacyisimportant** [bromite/bromite#98]
Why you have added and even recommed Cloudfare? It's like one of the worst companies for privacy. It's actively blocking TOR uses and it's very anti-privacy company. https://www.reddit.com/r/privacy/comments/41cb4k/be_careful_with_cloudflare/ and dor dns

**Comment by Madis0** [comment]
It is one of the first famous DNS over TLS providers. It is good to have choice though, if you can suggest more of those.

**Comment by privacyisimportant** [comment]
For **privacy** browser they shoud not be included. I suggest securedns.eu

**Comment by dimqua** [comment]
I agree, cloudflare and google servers shouldn't be used by default.
